README.md

```diff
@@ -68,18 +68,21 @@ working in Cloud Shell, you already have Python and `gcloud` installed.
 
 #### Set up the environment
 
-1. Run `gcloud auth login`, be sure to use the account that has permissions on
-   the Google Cloud project you are working on.
-2. Run `gcloud config set project PROJECT_ID`. Where PROJECT_ID is the project
+1. Run `gcloud config set project PROJECT_ID`. Where PROJECT_ID is the project
```

Comment:
This shouldn't be necessary on cloud shell; it will already be set to the current project.

Reply:
I've made a few changes, PTAL.